rust-runtime/aws-smithy-types/src/operation.rs

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//! Types for representing the interaction between a service an a client, referred to as an "operation" in smithy.
+//! Clients "send" operations to services, which are composed of 1 or more HTTP requests.
+
+/// Errors for operations
+pub mod error {
```

Comment:
This should go under the existing root-level `error` module: `crate::error::operation`

Reply:
That arrangement is better at the new home. I seem to have focused too much on preserving the original module path. Addressed in f6bb4ba.